### DISCUSSION TITLE
Contributing to Docs

### DISCUSSION THREAD

**Discussion by amirhmk** [InteractiveComputerGraphics/PositionBasedDynamics#128] [Q&A]
Hi there, 

I see that the XPBD module is not defined in the documentation. How does one go about updating the docs? I'm trying to implement a paper based on XPBD. Thanks!

**Comment by janbender** [comment]
Hi, 

The problem is that we often do not have enough time to write the docs. If you want to contribute here, you could write some doxygen comments and make a pull request.
